app/controllers/people_controller.rb

```diff
@@ -5,20 +5,22 @@ class PeopleController < ApplicationController
 
   def signin
     #This is the sign-in sheet, not anything about authentication
-    @people = Person.active
-    @department = Department.where(name: "Police")&.first
+    @department = Department.where(shortname: "BAUX").first
```

Comment:
This PR continues to hard-code strings into ReadyResponder ... including into the `signin`, `orgchart`, and `index` methods.  Would now be a good time to, instead, get away from such a practice?

I would have suggested considering whether this query should instead be a scope in the Person model, except that I'm unclear why we are hard-coding this query at all.